src/video/video.js

```diff
@@ -42,6 +43,9 @@ class Video extends Base {
     /** @type {LiveStreams} */
     this.LiveStreams = new LiveStreams(this);
 
+    /** @type {PlaybackIds} */
+    this.PlaybackIds = new PlaybackIds(this);
```

Comment:
nitpick, should it be `PlaybackIds` or `PlaybackIDs` ? I *think* the latter is canonical

Reply:
I agree that it _looks_ better, but I just went for camel case there

Reply:
@dylanjha Should I change this to `PlaybackIDs`

Reply:
I guess we already have used `PlaybackId` elsewhere like here: https://github.com/muxinc/mux-node-sdk/blob/master/src/video/resources/assets.js#L147

So we should be consistent, keep what you have

Reply:
👌